middlewares/sequelize-options.js

```diff
@@ -0,0 +1,20 @@
+module.exports = function (req, res, next) {
+    let where = req.query.where ? JSON.parse(req.query.where) : req.query
+
+    let options = {
+        where
+    }
+
+    if (req.query.populate) {
+        options.include = req.query.populate.split(',');
+        delete options.where.populate;
+    }
+
+    if (req.query.fields) {
+        options.attributes = req.query.fields.split(',');
+        delete options.where.fields;
+    }
+
```

Comment:
this two `if` statments could be isolated methods, and the `delete` operation is now useful when there is a `req.query.where`